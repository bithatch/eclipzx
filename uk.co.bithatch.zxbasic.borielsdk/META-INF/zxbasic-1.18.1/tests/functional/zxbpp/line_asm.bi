ASM
NOP
#line 25
END ASM
#warning this should be line 26
