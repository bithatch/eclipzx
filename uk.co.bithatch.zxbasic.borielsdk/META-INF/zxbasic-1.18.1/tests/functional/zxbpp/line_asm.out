#line 1 "line_asm.bi"
ASM
NOP
END ASM
